docs/extensions/get-started/overview.md

```diff
@@ -0,0 +1,20 @@
+# Extension Development Overview
+
+This is a general overview to how the development of an extension will procede. For building extensions there will be a few things that you should have installed, and some that might help.
+
+### Required:
+- [Node.js](https://www.nodejs.org/en/)
+- [Git](https://www.git-scm.com/)
+- Some sort of text editor, we recommend [VSCode](https://code.visualstudio.com/)
+- We use [Webpack](https://www.webpack.js.org/) for compilation. All extension need to be at least compatable with a webpack system.
+
+### Recommened
+- [Yarn](https://www.yarnpkg.com/), this is a better Command Line Interface (CLI) tool for downloading packages than `npm`
```

Comment:
Not sure if we need yarn recommendation (yet). Our samples are still using npm.

Reply:
Ah, I changed that in #1245 because all of our other commands (in root `package.json`) use yarn. I can remove it if you want though.

Reply:
@Nokel81 I'm not seeing that kind of changes in #1245?

Reply:
Ah, I think I misremembered thinking of changing it for actually changing it.

Reply:
Let's remove `yarn` recommendation for now (we can put it back when our own examples are using it).